exercises/error-handling/error_handling_test.go

```diff
@@ -27,6 +27,14 @@ func (mr mockResource) Close() error      { return mr.close() }
 func (mr mockResource) Frob(input string) { mr.frob(input) }
 func (mr mockResource) Defrob(tag string) { mr.defrob(tag) }
 
+// If this test fails and you've properly defined testVersion the requirements
```

Comment:
Yes, that was my thought also. Should we leave this here then, if it will be covered in `hello-world`?